lib/vdsm/tool/config_lvm_filter.py

```diff
@@ -122,6 +123,47 @@ def config_with_filter(
         return CANNOT_CONFIG
 
 
+def config_with_devices(args, mounts, current_wwids, wanted_wwids):
+
+    # Check if the lvm devices is already configured.
+    if lvmdevices.is_configured():
+        print("LVM devices already configured for vdsm")
+        return
+
+    # Before creating devices file we have to also configure multipath
+    # blacklist.
+
+    if current_wwids != wanted_wwids:
+        _print_wanted_blacklist(wanted_wwids)
+
+        if not args.assume_yes:
+            if not common.confirm("Configure host? [yes,NO] "):
+                return NEEDS_CONFIG
```

Comment:
Why confirm only if we have to configure multipath?

I would expect that configuring lvm devices will work exactly like
lvm filter:

1. Show current mounts
2. Show suggested configuration (e.g. we are going to configure lvm to use only these devices ...)
3. Show suggested changes in multipath if needed
4. Ask user confirmation
5. Configure